packages/@aws-cdk/aws-lambda-nodejs/parcel-bundler/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# runs the parcel-bundler npm package to package and install dependencies of nodejs lambda functions
+ARG NODE_TAG
+FROM node:${NODE_TAG}
+
+RUN yarn global add parcel-bundler@^1
```

Comment:
Is there not a prebuilt Parcel image up on DockerHub that we can use?

Reply:
Not really... also the idea is to maybe customize this image to address this https://github.com/aws/aws-cdk/issues/6323 and support externals (like aws-sdk).